RPT_Application/src/main/java/com/harman/report_database/Task.java

```diff
@@ -0,0 +1,107 @@
+package com.harman.report_database;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "task")
+public class Task {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+
+	private String type; // enum?
+	private String name;
+	private String placement; // enum?
+	private String description;
+	private WwbType wwbType;
+
+	@ManyToOne
+	private Report report;
+
+	public Task() {
+	}
+
+	public Task(String type, String name, String placement, String description, String wwbTypeString) {
```

Comment:
Why wwb is string not WwbType? If we really need conversion from string to enum it should be a method of enum class.

Reply:
fixed